platforms/ios/demo/TangramDemo.xcodeproj/project.pbxproj

```diff
@@ -411,7 +415,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = G436CJ835H;
```

Comment:
oops, I would prefer dev team string empty here. I know xcode added it automatically.

Reply:
Gah! Pay no attention to the DEVELOPMENT_TEAM behind the curtain!